### DISCUSSION TITLE
How can I see my transaction/expenses and budget category at the same time?

### DISCUSSION THREAD

**Discussion by EricTheMagician** [firefly-iii/firefly-iii#5657] [Firefly III] [answered]
Hi,

In the "transactions/expenses" page, is there a way to see which transaction belongs to which budget?
What I'm trying to achieve is to see if I mis-placed transactions in to the wrong budget. 

I mean, it doesn't need to be there specifically, but it would make sense to me for it to be in that page as there's a category option there and that page shows the budget pie chart at the top, but I don't see an option to enable it.

**Comment by EricTheMagician** [comment]
I found a solution. But it's super hidden, but it might end up as a feature request:
I have to select all my items in the expense and select the action "edit individually" and then I can see them all at once.
It's there and it will serve the purpose that I needed, but it might not be SO approved.

**Comment by JC5** [comment, marked as answer]
I'm working on a new layout, where I want to make columns selectable.

**Reply by EricTheMagician** [reply]
Awesome. Is there an issue that I can subscribe to in order to follow the development of it?
Anyways, thanks for the great software!
